Libraries/CommonFunctions.psm1

```diff
@@ -4284,4 +4284,106 @@ Function Remove-InvalidCharactersFromFileName
     $WindowsInvalidCharacters = [IO.Path]::GetInvalidFileNameChars() -join ''
     $Regex = "[{0}]" -f [RegEx]::Escape($WindowsInvalidCharacters)
     return ($FileName -replace $Regex)
+}
+
+function Get-KVPItem {
+	param (
+	    $VMName,
+	    $server,
+	    $keyName,
+		$Intrinsic
+	)
+
+    $vm = Get-WmiObject -Namespace root\virtualization\v2 -Query "Select * From Msvm_ComputerSystem Where ElementName=`'$VMName`'"
+    if (-not $vm)
+    {

```

Comment:
I know it means changes in a lot of places, but please stick with 
"if (check) {
} else {
}"

Reply:
moved else

Reply:
Moved else